test/data/templateWithImages.json

```diff
@@ -0,0 +1,18 @@
+{
+    "name": "imagestemplate",
```

Comment:
Since all templates should have a namespace now, can you add one to this example either by adding a `namespace` field or using `name: namespace/imagestemplate`?